src/server/debug.rs

```diff
@@ -963,6 +1024,102 @@ impl<ER: RaftEngine> Debugger for DebuggerImpl<ER> {
         self.reset_to_version_manager.start(version.into());
     }
 
+    fn key_range_flashback_to_version(
+        &self,
+        version: u64,
+        region_id: u64,
+        start_key: &[u8],
+        end_key: &[u8],
+        start_ts: u64,
+        commit_ts: u64,
+    ) -> impl Future<Output = Result<()>> + Send {
+        let store_id = self.get_store_ident().unwrap().get_store_id();
+        let r = self.region_info(region_id).unwrap();
+        let region = r
+            .region_local_state
+            .as_ref()
+            .map(|s| s.get_region().clone())
+            .unwrap();
+
+        let is_in_flashback = region.get_is_in_flashback();
```

Comment:
We should also check if the `flashback_start_ts` is the same, if not, the prepare flashback should be redone.

Reply:
When will meet this situation? start_ts has not changed since it was acquired once.